src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -2282,7 +2281,7 @@ private bool HasDescendantsWith(XamlObjectDefinition xamlObjectDefinition, Func<
 
 		private bool IsXamlTypeConverter(INamedTypeSymbol? symbol)
 		{
-			return _xamlConversionTypes.Any(ns => SymbolEqualityComparer.Default.Equals(ns, symbol));
+			return symbol?.GetAttributes().Any(a => a.AttributeClass?.Equals(_createFromStringAttributeSymbol, SymbolEqualityComparer.Default) == true) == true;
```

Comment:
I assume that is is more efficient to read the attributes for all types than building a map for all types?

Reply:
The idea is that now we don't read all types. Only do for types we think could have the attribute.